board-together/FE-Board-Together
# Spencer/feature/user dashboard

**NOTE**: Run `npm i` before proceeding. This merge adds an additional library for testing React components in Cypress.
#### What's this PR do?
This merge will add the following:
- General structure of `User Dashboard` component, including some CSS
- Dummy data JSON file for `user`
- Initial Cypress tests for `User Dashboard`

#### Where should the reviewer start?
- In CLI run `git fetch` and then `git checkout combined/feature/component-architecture`
- Look at code and file structure in IDE of preference
- In CLI run `npm start` which will automatically open the application in the browser.
#### How should this be manually tested?
- In the browser, inspect the page to see the general layout of the user dashboard
- In CLI run `npm run cypress`, and click on `user_dashboard` to see all passing tests.
#### Screenshots
![image](https://user-images.githubusercontent.com/74210902/217330296-2c887f89-e221-4bf3-b76f-48a0cec7445b.png)
<img width="1139" alt="image" src="https://user-images.githubusercontent.com/74210902/217330663-54c50bb2-a01f-41b4-8658-65aa35003ae0.png">
